packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/event-data-explorer/event-data-explorer-dialog/event-data-explorer-dialog.component.ts

```diff
@@ -96,16 +99,41 @@ export class EventDataExplorerDialogComponent {
     this.dialogRef.close();
   }
 
+  private async unzip(data: ArrayBuffer) {
```

Comment:
I've seen it, and I've tried to use it but here we expect a single file in the zip and we should warn if it's not the case. So was easier to do a different function

Reply:
Ok, I've tried to follow your line, but I do not manage to use the helper for reading zip files. It's in phoenix-event-display while we are here in phoenix-ui-component i nphoenix-ng. I do not think I have access to it.

Reply:
Oh, it would need to be exported to be exported from the library. I think the current implementation is fine as well. Thanks for trying my suggestion.